src/routes/demos/activate/.Step5MicrosoftIntegration.svelte

```diff
@@ -0,0 +1,129 @@
+<script lang="ts">
+  export let tenantId;
+  export let clientId;
+  export let privateKey;
+  export let clientCertificate;
+
+  const logo = 'https://cdn.jsdelivr.net/npm/simple-icons@v7/icons/microsoft.svg';
```

Comment:
$lib/constant maybe? if it's only being used here, your call.

Reply:
Currently, this is the only usage. I plan to add https://www.npmjs.com/package/simple-icons as a dependency.